lib/attr_encrypted.rb

```diff
@@ -160,11 +160,11 @@ def attr_encrypted(*attributes)
       end
 
       define_method(attribute) do
-        instance_variable_get("@#{attribute}") || instance_variable_set("@#{attribute}", decrypt(attribute, send(encrypted_attribute_name)))
+        instance_variable_get("@#{attribute}") || instance_variable_set("@#{attribute}", attr_encrypted_decrypt(attribute, send(encrypted_attribute_name)))
       end
 
       define_method("#{attribute}=") do |value|
-        send("#{encrypted_attribute_name}=", encrypt(attribute, value))
+        send("#{encrypted_attribute_name}=", attr_encrypted_encrypt(attribute, value))
```

Comment:
Calling `attr_encrypted(:secure_field)` is supposed to create two methods, `#secure_field` and `#secure_field=`. This ensures that those methods use the `attr_encrypted` implementations for decrypt/encrypt.

Otherwise, in rails 7 this would call the rails-versions of `#encrypt`/`#decrypt`?